### PR TITLE
Run more analyzers OOP

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/13838
Fixes https://github.com/dotnet/roslyn/issues/13639

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide @dpoeschl @heejaechang

File: src/Features/Core/Portable/UseAutoProperty/AbstractUseAutoPropertyAnalyzer.cs [hunk 1]
```diff
@@ -4,29 +4,34 @@
...
+        protected AbstractUseAutoPropertyAnalyzer()
+            : base(IDEDiagnosticIds.UseAutoPropertyDiagnosticId, s_title, s_title)
+        {
+        }
 
-        private readonly static DiagnosticDescriptor s_fadedTokenDescriptor = new DiagnosticDescriptor(
-            UseAutoPropertyFadedToken, FeaturesResources.Use_auto_property, FeaturesResources.Use_auto_property,
-            "Language", DiagnosticSeverity.Hidden, isEnabledByDefault: true,
-            customTags: new[] { WellKnownDiagnosticTags.Unnecessary });
+        //private readonly static DiagnosticDescriptor s_descriptor = new DiagnosticDescriptor(
```
Comment (heejaechang) [inline src/Features/Core/Portable/UseAutoProperty/AbstractUseAutoPropertyAnalyzer.cs]: ?

**Comment by heejaechang** [comment]
are you okay with running simplify type name for all files without addressing perf issue of simplify type name analyzer?

**Comment by CyrusNajmabadi** [comment]
test this please

**Pull request closed without merging**